modules/tinymce/src/plugins/table/test/ts/browser/ui/TableCellDialogTest.ts

```diff
@@ -23,23 +23,34 @@ UnitTest.asynctest('browser.tinymce.plugins.table.TableCellDialogTest', (success
   };
 
   let events = [];
-  const logEvent = (event: EditorEvent<{}>) => {
-    events.push(event);
+  const logEventTypes = (event: EditorEvent<{}>) => {
+    events.push(event.type);
+  };
+
+  let tableModifiedEvents = [];
+  const logTableModified = (event: EditorEvent<TableEventData>) => {
+    tableModifiedEvents.push(event);
   };
 
-  const sClearEvents = Step.sync(() => events = []);
+  const sClearEvents = Step.sync(() => {
+    events = [];
+    tableModifiedEvents = [];
+  });
+
```

Comment:
Double blank line here. I found an eslint rule for this yesterday that we should turn on at some stage: https://eslint.org/docs/rules/no-multiple-empty-lines

Reply:
How do we feel about adding that rule in this PR?

Reply:
There's probably a number of existing errors, so I'd prefer to do it separately. It also really should end up in the `@tinymce/eslint-plugin` instead.